contrib/firebaseio/src/main/java/com/google/cloud/dataflow/contrib/firebase/io/FirebaseSource.java

```diff
@@ -366,14 +376,16 @@ protected Firebase getRef(){
 
   @Override
   public void validate() {
-
     AuthData result;
-
     try {
       result = auther.authenticate(this.getRef());
-    } catch (FirebaseException|InterruptedException e) {
+    } catch (FirebaseException e) {
+      result = null;
+      logger.error("Firebase authentication error", e);
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
+      logger.error("Authenticator interrupted", e);
       result = null;
-      e.printStackTrace();
     }
     Preconditions.checkNotNull(result);
```

Comment:
maybe provide a good error message here? What should a user do to debug this / what is the recourse if this fails?